src/NuGet.Core/NuGet.CommandLine.XPlat/CommandOutputLogger.cs

```diff
@@ -91,7 +93,26 @@ private void LogInternal(LogLevel logLevel, string message)
             {
                 caption = logLevel.ToString().ToLowerInvariant();
             }
-            Console.WriteLine($"{caption}: {message}");
+            
+            Console.Write(PrefixCaption(caption, message));
+        }
+
+        private static string PrefixCaption(string caption, string message)
```

Comment:
Maybe it could do a message.IndexOf('\n') check to noop for the majority of messages